blog.html

```diff
@@ -10,11 +10,11 @@
 <body>
 <header>
     <div id="nav">
-        <a href="#" class="abc" >Ling Blog</a>
-        <a href="#" class="abc" >Home</a>
-        <a href="#" class="abc" >About</a>
-        <a href="#" class="abc" >Portfolio</a>
-        <a href="#" class="abc" >Tags</a>
+        <span style='float:left'><a href="#" class="abc">Ling Blog</a></span>
```

Comment:
Looks nice. Just one minor suggestion: normally we would like to put only one tag in one line. e.g. 
<span style='float: left'>
    <a href="#" class="abc">Ling Blog</a>
</span>